README.md

```diff
@@ -354,38 +354,24 @@ cargo doc --open
 
 # Container
 
-If you want to, you can also use a container image.
+If you want to, you can also use a container image with [Docker][docker], [Podman][podman] or any other container tool.
 However, we recommend using the native binary if possible.
 
-## Build
-
-You can either use the provided script in the following way to build the image and also interact with it
-
-```
-./ddnnife_dw.sh
-```
+For an overview, see: https://github.com/SoftVarE-Group/d-dnnf-reasoner/pkgs/container/ddnnife
 
-or you can build the image directly by hand.
-Here, ```ddnnife``` is the name of the image.
+There is a tag for the main branch and for each tagged release, the latest one is `latest`.
+To pull the latest tagged version, use:
 
 ```
-docker build -t ddnnife .
+docker pull ghcr.io/softvare-group/ddnnife:latest
```

Comment:
It will be available once we release a tagged version. There is currently one for this branch but there will be no new branch containers, only main and tags will be built.